modules/calib3d/src/chessboard.cpp

```diff
@@ -611,37 +703,41 @@ void FastX::detectImpl(const cv::Mat& _gray_image,
     CV_CheckTypeEQ(_gray_image.type(), CV_8UC1, "Unsupported image type");
 
     // up-sample if needed
-    cv::Mat gray_image;
-    int super_res = int(parameters.super_resolution);
+    cv::UMat gray_image;
```

Comment:
I though we want to get rid of all implicit GPU calls in algorithms' internals like we did with CascadeClassifier.